samples/python/greedy_causal_lm/greedy_causal_lm.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import openvino_genai
+
+
+def streamer(subword):
+    print(subword, end='', flush=True)
+    return False
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('model_dir')
+    parser.add_argument('prompt')
+    args = parser.parse_args()
+
+    device = 'CPU'  # GPU can be used as well
+    pipe = openvino_genai.LLMPipeline(args.model_dir, device)
+
+    config = openvino_genai.GenerationConfig()
```

Comment:
```suggestion
    config = openvino_genai.GenerationConfig(max_new_tokens=100)
```
is it possible to write this way?

Reply:
No. Only generate() can do that.

Reply:
can we support it?

Reply:
No. I believe we should finalize the current sate for the release first.

Reply:
ok, let's have it as future improvement